todo_app/data/trello_items.py

```diff
@@ -0,0 +1,121 @@
+from flask import session
+import requests, json
+
+# Define a class for Card type 
+class Item:
+        def __init__(self, id, name, statusid, status = 'To Do'):
+            self.id = id
+            self.name = name
+            self.statusid = statusid
+            self.status = status
+
+'''   
+        @classmethod
+        def from_trello_card(cls, id, name, list):
+           return cls( id, name, list )
+'''
```

Comment:
If we're not going to use this, we shouldn't bother keeping it here commented out!